### DISCUSSION TITLE
i got error about web socket on tab live calls

### DISCUSSION THREAD

**Discussion by tanapatMintel** [linkerd/linkerd2#10756] [Help]
i use gke and i got error like picture below
<img width="1440" alt="Screen Shot 2566-04-16 at 03 33 31" src="https://user-images.githubusercontent.com/66985646/232252122-364292e2-d182-4759-9641-da8568c35cb6.png">

**Comment by Kukkui** [comment]
I've got the same problem!

**Comment by Rattapong-ICE** [comment]
I ran into this problem too.

**Comment by flookvatchara** [comment]
i met too
+1

**Comment by mejling-pol** [comment]
This problem, I encountered the same.

**Comment by knightdevz** [comment]
I ran into this problem too.

**Comment by JameApisit** [comment]
i got this problem too.

**Comment by alpeb** [comment]
It appears this has been a long-standing issue with GKE. I can reproduce it in both 2.12.4 and 2.13.1. Does exposing the dashboard via ingress as detailed in the [docs](https://linkerd.io/2.13/tasks/exposing-dashboard/) serve as a workaround? Or perhaps what's discussed in #5897?
